docs/tutorials/tut-integration-ui.md

```diff
@@ -190,10 +227,10 @@ class Client(BaseClient):
         return self._http_request(method='POST', url_suffix='yoda', data={'text': text}, resp_type='json',  ok_codes=(200,))
 ```
 
-The Client is an object that communicates with the API. We create a class called *Client*.
-When a Client object is created, it instantiates a parent *BaseClient* using the params we have set up (whether to use proxy, whether to allow insecure connections, and the base URL). If the user provided values to the *api_key* parameter, the Client sets the headers it will use accordingly. 
+The Client is an object that communicates with the API. You create a class called *Client*.
+When a Client object is created, it instantiates a parent *BaseClient* using the params you have set up (whether to use proxy, whether to allow insecure connections, and the base URL). If the user provides values to the *api_key* parameter, the Client sets the headers it will use accordingly. 
```

Comment:
```suggestion
When a Client object is created, it instantiates a parent *BaseClient* using the parameters you have set up (whether to use proxy, whether to allow insecure connections, and the base URL). If the user provides values to the *api_key* parameter, the Client sets the headers it will use accordingly. 
```